### PR TITLE
Lesson 4 2

### PR DESCRIPTION
**Pull request by aboubacar108**
added new elements in index.html file.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const copyright = document.createElement('p');
+copyright.innerHTML = `<span>Aboubacar Kaba ${thisYear}</span>`;
+footer.appendChild(copyright);
+
+const skills = ["Internal & External Audit","Digital Audit using Analytics", "SOX", "IFRS","Financial Analysis & Reporting", "Risk assessment", "Report writing", "Customer Service", "Web Development", "Programming with JavaScript, HTML, CSS", "Advanced Excel" ];
+const skillsSection = document.getElementById('skills');
+const skillsList = document.querySelector('.list');
+for (let i = 0; i < skills.length; i++) {
+    const skill = document.createElement('li');
+    skill.innerHTML = skills[i];
```
Comment (jeremybohannon) [inline js/index.js:13]: innerText is safer but innerHTML works as well

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+let today = new Date();
+let thisYear = today.getFullYear();
+const footer = document.querySelector('footer');
+const copyright = document.createElement('p');
+copyright.innerHTML = `<span>Aboubacar Kaba ${thisYear}</span>`;
+footer.appendChild(copyright);
+
+const skills = ["Internal & External Audit","Digital Audit using Analytics", "SOX", "IFRS","Financial Analysis & Reporting", "Risk assessment", "Report writing", "Customer Service", "Web Development", "Programming with JavaScript, HTML, CSS", "Advanced Excel" ];
+const skillsSection = document.getElementById('skills');
+const skillsList = document.querySelector('.list');
```
Comment (jeremybohannon) [inline js/index.js:10]: you can query for classNames with `getElementsByClassName`

**Review by jeremybohannon — APPROVED** [review]
LGTM - looks good to me

**Comment by jeremybohannon** [comment]
Got some merge conflicts but code looks good

File: js/index.js (hunk 1 above)
Comment (aboubacar108) [reply js/index.js:13]: OK

**Comment by aboubacar108** [comment]
Review completed. Merge conflicts persist.

**Pull request closed without merging**